src/routes/choice-exercises/components/SingleChoiceExercise.tsx

```diff
@@ -70,6 +70,10 @@ const ChoiceExerciseScreen = ({
   const documents = newDocuments ?? data
   const currentDocument = documents ? documents[currentWord] : null
 
+  const result = results.find(result => result.id === currentDocument?.id)
+  const nthRetry = !result ? 0 : result.numberOfTries
+  const needsToBeRepeated = nthRetry < numberOfMaxRetries && (!result || result.result === SIMPLE_RESULTS.incorrect)
```

Comment:
Shouldn't it be repeated if it is nearly correct? Or is this handled somewhere else?

Reply:
Nearly correct is not possible for SingleChoiceExercises